js/script.js

```diff
@@ -0,0 +1,57 @@
+console.log("ready")
+$('#stop').hide();
+$(document).ready(function(){
+
+	$('#search').on('click', function(e) {
+	 	e.preventDefault();
+	var pics = [];
+	var counter = 0;
+
+
+//hide search field and show Reset button
+    $('#container').hide(750);
+    $('#stop').show(100);
+
+
+//search reddit posts for input term
+$.get('https://www.reddit.com/search.json?q=' + $('#searchTerm').val()).done(function(data) 
+	{ 
+
+		console.log(data);
+		pics = data; 
+
+	//filter for only posts with images
+	var filtered = pics.data.children.map(function (child) {
```

Comment:
Your images are small because you're grabbing the thumbnails. Look through your data more and try to find a better place to grab images from.